dev-libs/libinput.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+
+import stdlib
+from stdlib.template import autotools
+from stdlib.manifest import manifest
+
+
+@manifest(
+    name='libinput',
+    category='dev-libs',
+    description='''
+    A library that handles input devices for display servers and other applications that need to directly deal with input devices.
+    ''',
+    tags=['input', 'mouse', 'keyboard'],
+    maintainer='grange_c@raven-os.org',
+    licenses=[stdlib.license.License.MIT, stdlib.license.License.CUSTOM],
+    upstream_url='https://bitmath.org/code/mtdev/',
+    kind=stdlib.kind.Kind.EFFECTIVE,
+    versions_data=[
+        {
+            'semver': '1.1.5',
```

Comment:
This version does not seem to match the one from the archive URL below, is it normal ?

Reply:
I may or may not have been half-asleep when writting this shit. Thanks.